app/models/vmdb_database_setting.rb

```diff
@@ -4,7 +4,7 @@ class << self
   end
   self.aar_columns = []
 
-  self.table_name = 'pg_settings'
+  self.table_name = 'pg_catalog.pg_settings'
```

Comment:
:bug:

This model had an issue (failing to find its column types, IIRC) without the schema qualification... but at least one of the others broke *with* the schema. I suspect we have an edge case with the semi-magical `pg_catalog` schema.